packages/relay/index.js

```diff
@@ -53,7 +53,7 @@ export type RelayPaginationProp = {|
 |};
 
 type RefetchOptions = {|
-  +force?: boolean,
+  +force: true, // we always have to force the refetch since we use offline cache (TODO: wrap it)
```

Comment:
So because you use offline cache, you want to display first data from the cache (unless `cacheConfig.force` is true), and then fetch data from the network, and for it to update, this is necessary. Correct?

Reply:
It's only because of UX let's say. Currently, we try to load data from cache first and after that load them from API and only update the components if necessary (new data).

But this looks weird when refreshing - the loader stops spinning very fast because we read it first from the cache (and that's fast). Therefore I wanted to really explicitly say "re-fetch" and this way we don't even touch the cache while refreshing (and the spinner is really snipping until we re-fetch it from the network). We should wrap it later so it's not necessary to write it everywhere...